completion.sh.hbs

```diff
@@ -19,10 +19,10 @@ _yargs_completions()
 
     # if no match was found, fall back to filename completion
     if [ ${#COMPREPLY[@]} -eq 0 ]; then
-      COMPREPLY=( $(compgen -f -- "${cur_word}" ) )
+      COMPREPLY=()
```

Comment:
Ooh, I like the result of this PR. @tschaub, can you provide any context on the initial draft of this conditional?

Reply:
Hmm.  Having trouble recalling the motivation.  If `()` is a better way to fall back to filename completion, then this looks like the right change.

Reply:
@tschaub Thanks for taking a look!